src/dlgNotepad.cpp

```diff
@@ -42,9 +42,30 @@ dlgNotepad::dlgNotepad(Host* pH)
 {
     setupUi(this);
 
+    //: label for prepended text entry box in notepad
+    label_prependText = new QLabel("Prepend", this);
```

Comment:
```suggestion
    label_prependText = new QLabel(tr("Prepend"), this);
```

Please make sure all strings are either wrapped in tr() if visible to users, so they can be translated , or qsl() instead if they deliberately should not be translated.